src/tic_tac_toe/user_interface.clj

```diff
@@ -2,14 +2,16 @@
   (:use [clojure.math.numeric-tower :only [sqrt]]
         [clojure.string :only [join]])
   (:require [tic-tac-toe.io :as io]
-            [tic-tac-toe.messenger :as messenger]
+            [tic-tac-toe.announcer :as announcer]
             [tic-tac-toe.presenter :as presenter]))
 
 (defprotocol UI
   (update-display [this message])
   (get-input [this])
   (clear-ui [this])
-  (prompt-move [this board])
+  (flush-ui [this])
```

Comment:
Is `flush` a concern of the UI?

Reply:
[Flush](https://github.com/saralein/tic-tac-toe_clojure/pull/4/files#diff-6fc8428f8d5a98bc15e079cfdc7c2346R9) should now just be a concern of the IO.